lib/gmp/gmp.nit

```diff
@@ -0,0 +1,336 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Multi precision integer and rational number using gmp lib
+module gmp
+
+private import native_gmp
+
+redef class Numeric
+
+    # The BigInt equivalent of `self`
+    fun to_bi: BigInt do return self.to_i.to_bi
```

Comment:
I understand your point, still It seems to me that `BigInt` lacks a

~~~nit
redef fun to_bi do return self
~~~

`Ratio` too, unless I missed it